### PR TITLE
[s] just gotta silently relocate this

### PR DESCRIPTION
**Pull request by Iamgoofball**
My boy, this peace is what all true maintainers strive for.

:cl: Iamgoofball
add: adds a flying carpet to ride
/:cl:

### CODE REVIEW

**Comment by 81Denton** [comment]
mah boi

**Comment by nervere** [comment]
@Dorsisdwarf How do you feel now? He has no limits!

**Comment by ExcessiveUseOfCobblestone** [comment]
I can show you a world....

File: _maps/shuttles/emergency_carpet.dmm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"a" = (/turf/template_noop,/area/template_noop)
```
Comment (astralloop) [inline _maps/shuttles/emergency_carpet.dmm]: tgm

**Comment by Kromgar** [comment]
What happened to mapdiff bot i wanted to see what it looks like

File: code/datums/shuttles.dm [hunk 2]
```diff
@@ -199,6 +199,12 @@
 	description = "A small, but feature complete shuttle. It boasts a card table to keep crew members occupied on the long flight home."
 	credit_cost = 1000
 
+/datum/map_template/shuttle/emergency/carpet
+	suffix = "carpet"
+	name = "Magic Carpet"
+	description = "Shadala, we're off!"
```
Comment (Armhulen) [inline code/datums/shuttles.dm]: squadala

**Pull request closed without merging**